src/main/webapp/src/tools.html

```diff
@@ -90,5 +90,22 @@ <h2>Hiero</h2>
             </div>
         </div>
     </div>
+    
+    <div class="one-of-two">
+        <div class="well tools-well">
+            <h2>PolygonCreator</h2>
+            <hr>
+            <p>A tool for creating vertices for your polygon object.</p>
+
+            <div class="container">
+                <img src="https://imagr.eu/up/vsKRr_example.png" />
+            </div>
+            <hr>
+            <div class="links">
+                <a href="https://github.com/kdenzel/PolygonCreator/releases/download/v1.1/PolygonCreator.jar">Download</a>
```

Comment:
Maybe just link the releases page? If you link it like this, users won't download newer versions like 1.2 because the link downloads 1.1 everytime

Reply:
Hi SimonIT, that is a good idea. Shall i change it in a new commit and make a new PR or how do we proceed?